### PR TITLE
removed string conversion and fabs deprecated warnings

### PR DESCRIPTION
**Pull request by obitodarky**
replaced deprecated function fabs by abs
also added string type to remove warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Amahi iOS App, new from scratch, in Swift.
 
 ## Requirements
 
-- iOS 9.0+
-- Xcode 9.0+
+- iOS 11.0+
+- Xcode 10.0+
```
Comment (codedentwickler) [inline README.md]: Why did you change this?

Comment (obitodarky) [reply README.md]: I changed setter for `statusBarStyle` by overriding `preferredStatusBarStyle` since `statusBarStyle` was deprecated in previous version of ios. 
I thought doing that won't work in the old ios versions. It seems like I may have been wrong. I will revert to old previous commit. Pardon my mistake.

**Comment by cpg** [comment]
Please resolve conflicts after I changed to base branch `beta`.

We work on the `beta` branch first, then later we push to mater and we release publicly.

File: README.md (hunk 1 above)
Comment (codedentwickler) [reply README.md]: Okay. Instead you can check for the specific version of ios and use the deprecated code on old version and the lastest on the new.

Something like this

```
if #available(iOS 9.0, *) {
    // use the feature only available in iOS 9
    // for ex. UIStackView
} 
else {
    // or use some work around
}
```

Comment (obitodarky) [reply README.md]: Thanks for the tip.
I've used a different method since `#available` was giving me errors 

This is what I have come up with 
`        let systemVersion = UIDevice.current.systemVersion
        
        if((Double)(systemVersion) >= 10.0){
            override var preferredStatusBarStyle: UIStatusBarStyle { return .lightContent}
        } else {
            UIApplication.shared.statusBarStyle = .lightContent
        }`
